### PR TITLE
start/stop/reboot RDS instance

### PR DESCRIPTION
**Pull request by root-beer**
- [x] 開始/停止/再起動要求を行った結果の表示
- [x] ステータスの自動更新（フロントエンド側）
- [x] ステータスの starting or stopping がわかる表示
- [x] スケール変更時の表示を適切なものにする

### CODE REVIEW

File: app/controllers/infrastructures_controller.rb [hunk 1]
```diff
@@ -346,6 +346,27 @@ def save_schedule
...
+    rds = @infrastructure.rds(physical_id)
+    result = rds.start_db_instance
+    render text: result, status: 200
+  end
+
+  def stop_rds
+    physical_id = params.require(:physical_id)
+    rds = @infrastructure.rds(physical_id)
+    result = rds.stop_db_instance
+    render text: result, status: 200
```
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[brakeman]__
* Unescaped model attribute rendered inline [:ref](http://brakemanscanner.org/docs/warning_types/cross_site_scripting/)

File: app/controllers/infrastructures_controller.rb [hunk 2]
```diff
@@ -346,6 +346,27 @@ def save_schedule
     render text: I18n.t('operation_scheduler.msg.saved'), status: 200 and return
   end
 
+  def start_rds
+    physical_id = params.require(:physical_id)
+    rds = @infrastructure.rds(physical_id)
+    result = rds.start_db_instance
+    render text: result, status: 200
```
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[brakeman]__
* Unescaped model attribute rendered inline [:ref](http://brakemanscanner.org/docs/warning_types/cross_site_scripting/)

File: app/controllers/infrastructures_controller.rb [hunk 3]
```diff
@@ -346,6 +346,27 @@ def save_schedule
...
+    rds = @infrastructure.rds(physical_id)
+    result = rds.stop_db_instance
+    render text: result, status: 200
+  end
+
+  def reboot_rds
+    physical_id = params.require(:physical_id)
+    rds = @infrastructure.rds(physical_id)
+    result = rds.reboot_db_instance
+    render text: result, status: 200
```
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[brakeman]__
* Unescaped model attribute rendered inline [:ref](http://brakemanscanner.org/docs/warning_types/cross_site_scripting/)

File: app/controllers/infrastructures_controller.rb [hunk 4]
```diff
@@ -18,7 +18,7 @@ class InfrastructuresController < ApplicationController
   before_action :infrastructure_exist, only: [:show, :edit, :update, :destroy, :delete_stack, :stack_events]
 
 
-  before_action :set_infrastructure, only: [:show, :edit, :update, :destroy, :delete_stack, :stack_events, :show_rds, :show_elb]
+  before_action :set_infrastructure, only: [:show, :edit, :update, :destroy, :delete_stack, :stack_events, :show_rds, :show_elb, :start_rds, :stop_rds, :reboot_rds]
```
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[rubocop]__
* Style/SymbolArray: Use `%i` or `%I` for an array of symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-i)

**Comment by skyarch-networks** [comment]
Click here if you want to create rubocop's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/31754199/issues/pull_request/418/diff/26edf22f41eed975f77bbfaef5c4eeba875777ae?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

File: app/controllers/infrastructures_controller.rb (hunk 4 above)
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[RuboCop]__

Style/SymbolArray: Use `%i` or `%I` for an array of symbols.

File: app/controllers/infrastructures_controller.rb (hunk 2 above)
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[Brakeman]__

Unescaped model attribute rendered inline

File: app/controllers/infrastructures_controller.rb (hunk 1 above)
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[Brakeman]__

Unescaped model attribute rendered inline

File: app/controllers/infrastructures_controller.rb (hunk 3 above)
Comment (skyarch-networks) [inline app/controllers/infrastructures_controller.rb]: __[Brakeman]__

Unescaped model attribute rendered inline

File: app/controllers/infrastructures_controller.rb (hunk 4 above)
Comment (root-beer) [inline app/controllers/infrastructures_controller.rb]: > Style/SymbolArray: Use `%i` or `%I` for an array of symbols.
> [link](https://sideci.com/gh/31754199/pull_requests/418)

**Pull request merged**